tensorrt_llm/_torch/autotuner.py

```
@@ -1170,6 +1260,253 @@ def _profile_runners(
+        for tactic in tactics:
+            try:
+                time_measured = self._profile_single_kernel(
+                    runner=runner,
+                    inputs=input_tensors,
+                    tactic=tactic,
+                    tuning_config=tuning_config,
+                    use_cuda_graph=tuning_config.use_cuda_graph,
+                    **kwargs,
+                )
+                results.append((tactic, time_measured, None))
+            except Exception as e:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)